src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -223,6 +229,30 @@ protected void VerifyCurrentSignature(Signature expectedSignature)
             Assert.Equal(expectedSignature, currentSignature);
         }
 
+        protected void VerifyCurrentSignature(string content)
```

Comment:
That helper requires a lot of boilerplate--I'm going to remove it and replace the sole usage (which looks like this, BTW):
```
var expectedParameter = new Parameter
            {
                Name = "args"
            };

            var expectedSignature = new Signature
            {
                Content = "void Class1.Main(string[] args)",
                CurrentParameter = expectedParameter,
                Parameters = new[] { expectedParameter },
                PrettyPrintedContent = "void Class1.Main(string[] args)"
            };
```